spec/services/invoices/payments/stripe_service_spec.rb

```diff
@@ -49,6 +54,9 @@
         .and_return(provider_customer_service)
       allow(provider_customer_service).to receive(:check_payment_method)
         .and_return(provider_customer_service_result)
+
+      stub_request(:get, "https://api.stripe.com/v1/customers/#{stripe_customer.provider_customer_id}").
```

Comment:
Layout/DotPosition: Place the . on the next line, together with the method name. (https://rubystyle.guide#consistent-multi-line-chains)